tests/MPIParallelAutoPasTests/tests/utils/AutoPasConfigurationCommunicatorTest.cpp

```diff
@@ -0,0 +1,219 @@
+/**
```

Comment:
shortly explain each test:
what are you testing and why do you expect, what you checking.